shared/domains/chelonia/internals.js

```diff
@@ -488,6 +488,7 @@ export default (sbp('sbp/selectors/register', {
           method: 'POST',
           body: entry.serialize(),
           headers: {
+            ...headers,
```

Comment:
The line below that reads:

> 'Authorization': 'gi TODO - signature - if needed here - goes here'

Maybe now is a good time to remove it? 🤔 

We have signatures working now, and they've not using headers